scripts/utils.js

```diff
@@ -119,7 +119,7 @@ export function getPluralForm(number, forms) {
 
 
 export async function openBackgroundTab(url) {
-    let tab = await browser.tabs.getCurrent();
+    let [tab] = await browser.tabs.query({active: true, currentWindow: true});
```

Comment:
I'm unfamiliar with this syntax. What's the brackets signify?

Reply:
Assigning the first element of the right hand side array directly to `tab` (the same as `let tab = (await browser.tabs.query({active: true, currentWindow: true}))[0];`).